### ISSUE TITLE
CQUI Tips overlay

### ISSUE THREAD

**Issue by chaorace** [CQUI-Org/cqui#529]
CQUI has a lot of smaller and, sometimes, obscure features. Off the top of my head: right clicking the action button to skip blockers and middle clicking diplo banners to view other leaders' opinions come to mind.

I think this may be best implemented as a notification group in the new interface we're working on in PR #514, since this would be rather unobtrusive and we're planning to allow specific notification groups to be muted.

**Comment by bolbass** [comment]
Hi! As for me middle clicking is not the best choice for my mouse and I think I'm not alone :)

**Comment by chaorace** [comment]
@bolbass That's already in-game, this feature is about introducing a tip box for explaining shortcuts like these. For example, the tip box for the opinion feature can mention that one could also hold ctrl to achieve the same effect if the middle mouse button is not preferred, so as to prevent people from asking for features that are already implemented 😉

**Comment by bolbass** [comment]
Yeah, now I tried middle-click on leaders and was surprised with this feature :)

**Comment by Heinous-Hat** [comment]
That function also works if you hold ctrl as you mouse over the leader portrait. You have to be holding the key _before_ the cursor touches touches the portrait though. Hold ctrl and run the mouse cursor down the line of leader portraits and you'll see what I mean.

**Comment by chaorace** [comment]
And I quote...
# "For example, the tip box for the opinion feature can mention that one could also hold ctrl to achieve the same effect if the middle mouse button is not preferred"

**Comment by Heinous-Hat** [comment]
wait... let me get my glasses :-)

Missed that.
